src/main/utils/scanUtils.ts

```diff
@@ -13,7 +13,7 @@ export class ScanUtils {
         await vscode.window.withProgress(
             <vscode.ProgressOptions>{
                 location: vscode.ProgressLocation.Notification,
-                title: credentialsSet ? 'Xray Scanning' : 'GoCenter Scanning',
```

Comment:
it is already mentioned here :
https://github.com/jfrog/jfrog-vscode-extension/pull/34/files#diff-cec219d597d5f28de68dff0dbe8b630eR108 
so it will show as two times